Zend/zend_call_stack.c

```diff
@@ -611,7 +606,11 @@ static bool zend_call_stack_get_netbsd_vm(zend_call_stack *stack)
 static bool zend_call_stack_get_netbsd(zend_call_stack *stack)
 {
 	if (syscall(SYS__lwp_self) == 1) {
-		return zend_call_stack_get_netbsd_vm(stack);
+		void *ptr;
+		bool r;
+		r = zend_call_stack_get_netbsd_vm(stack, &ptr);
+		free(ptr);
```

Comment:
`ptr` is uninitialized if zend_call_stack_get_netbsd_vm returns before the malloc call